frontend/src/tests/UserDomainSelect.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { it, describe, beforeEach, expect } from 'vitest';
+import { mount } from '@vue/test-utils';
+import { UserType } from 'fam-app-acsctl-api';
+import type { VueWrapper } from '@vue/test-utils/dist/vueWrapper';
+import type { DOMWrapper } from '@vue/test-utils/dist/domWrapper';
+import UserDomainSelect from '@/components/grantaccess/form/UserDomainSelect.vue';
+
+describe('UserDomainSelect', () => {
+    let wrapper: VueWrapper;
+    let emitChange: unknown[][] | undefined;
+
+    let becidRadioBtn: DOMWrapper<HTMLInputElement>;
+    let idirRadioBtn: DOMWrapper<HTMLInputElement>;
+
+    beforeEach(async () => {
+        wrapper = mount(UserDomainSelect);
+        becidRadioBtn = wrapper.find('#becidSelect');
+        idirRadioBtn = wrapper.find('#idirSelect');
+
+    });
+
+    it('Should check the radioBtn box when clicked', async () => {
+
+        await becidRadioBtn.trigger('click');
+        expect(becidRadioBtn.element.checked).toBeTruthy();
```

Comment:
To be safe (since last time seems some false-positive result on expect). Could you also add `toBeFalsy()` on the other button check? For example, since it is clicked on 'bceid' and truthy, also expect 'idir' is falsy. 
Thanks.

Reply:
sure!